R/extract_spatial.R

```diff
@@ -97,8 +96,10 @@ feature_states.Spatial <- function(sp){
 #' @importFrom maptools map2SpatialPolygons
 #' @importFrom sp CRS
 get_states <- function(){
-  usa <- map("state", fill = TRUE, plot = FALSE)
-  IDs <- sapply(strsplit(usa$names, ":"), function(x) x[1])
```

Comment:
ok, I guess it is trickier than that. I don't think it is possible to combine the different maps in the way that you have coded it here @berdaniera